### DISCUSSION TITLE
FIXMEs and TODOs

### DISCUSSION THREAD

**Discussion by Lord-Kamina** [performous/performous#771] [General]
There are several of these peppered all around the code. Some are relatively innocent things like still using the C rand functions but there are also several pointing to issues with the gameplay itself or handling of certain things.

I think we should try to at least look at them and see if any of us can address these (for example, there's something relating to polyphony in the guitargraph)

**Comment by Baklap4** [comment]
See also: https://github.com/performous/performous/issues/261
